transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollStreamChannel.java

```diff
@@ -1038,7 +1033,7 @@ void epollInReady() {
                 pipeline.fireChannelReadComplete();
 
                 if (close) {
-                    shutdownInput();
+                    shutdownInput(false);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Prefix this call to "shutdownInput" with "super.". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2388)